vendor/bat-native-ads/src/bat/ads/internal/database/tables/categories_database_table.cc

```diff
@@ -64,6 +62,11 @@ void Categories::Migrate(
       break;
     }
 
+    case 3: {
```

Comment:
Did we skip 2? And should we have `MigrateToV1` , ``MigrateToV2`, and `MigrateToV3` as chained calls?

Reply:
Discussed and agreed